### ISSUE TITLE
test "persistent volume retains data" fails on Ubuntu node

### ISSUE THREAD

**Issue by okartau** [intel/pmem-csi#434]
A recently added test "persistent volume retains data" fails on Ubuntu node where a ssh login emits welcome banner, with expected data "hello" only shown after the banner, so we see test failure:

     persistent volume retains data
     go/src/github.com/intel/pmem-csi/test/e2e/storage/sanity.go:521

      read failure
      Expected
          <string>: Welcome to Ubuntu 19.04 (GNU/Linux 5.0.7-050007-generic x86_64)\n\n * Documentation:  https://help.ubuntu.com\n * Management:     https://landscape.canonical.com\n * Support:        https://ubuntu.com/advantage\n\n  System information as of Wed Oct 16 11:11:08 UTC 2019\n\n  System load:  0.44              Users logged in:        0\n  Usage of /:   7.6% of 48.29GB   IP address for ens5:    172.17.0.3\n  Memory usage: 19%               IP address for docker0: 172.18.0.1\n  Swap usage:   0%                IP address for cni0:    10.244.3.1\n  Processes:    149\n\n\n75 updates can be installed immediately.\n40 of these updates are security updates.\n\nFailed to connect to https://changelogs.ubuntu.com/meta-release. Check your Internet connection or proxy settings\n\n\nhello
      to equal
          <string>: hello

Hopefully there is way to get the file contents without getting ssh session info into buffer.

**Comment by pohly** [comment]
Ubuntu shouldn't treat command execution the same way as executing a login or interactive shell. Is it possible to suppress that message, perhaps with ssh command line options?

**Comment by okartau** [comment]
I tried to simplify.
The current method of running just ssh.N wrapper first, is equal to opening interactive shell in one command, then feeding target command into Stdin, right.
I propose to prepare command and run it in one exec call, being equal to run single "ssh cmd" command.
That seems also to get around login banner on Ubuntu.
I tested this in local Ubuntu-based tester and it works.
Created #439 to see results in CI.
If that works, we can consider  replacing also file creation part with single-cmd method, then can reuse the sshcmd setup part (which is repeated code already in existing code).
